.github/workflows/cd.yml

```diff
@@ -27,6 +27,10 @@ jobs:
         with:
           workload_identity_provider: ${{ vars.GCP_WORKLOAD_IDENTITY_PROVIDER }}
           service_account: ${{ vars.GCP_SERVICE_ACCOUNT }}
+      - name: Set up Node.js
```

Comment:
this is not actually needed, what I meant is that we don't specify a runtime in the deploy-cloudrun action below, so I guess it defaults to either what's the latest LTS they support, probably adhering to the engines field in package.json. maybe there's nothing we need to do here actually, just worth checking what version we're running on atm and making sure that when this is merged and deployed, it runs on a compatible node version